stdlib/logging/__init__.pyi

```diff
@@ -246,7 +246,7 @@ NOTSET: Final = 0
 class Handler(Filterer):
     level: int  # undocumented
     formatter: Formatter | None  # undocumented
-    lock: threading.Lock | None  # undocumented
+    lock: threading.RLock | None  # Corrected type
```

Comment:
I don't think the comment needs to be changed.

Reply:
@InFiNiTy0639 you marked Jelle's comment as "resolved" here but it doesn't look like you addressed his feedback